.github/ISSUE_TEMPLATE/bug-report-advanced

```diff
@@ -0,0 +1,74 @@
+name: "\U0001F41E Bug report"
+description: Create a report to help us improve
+title: "[Bug]: "
+labels: ["Triage"]
```

Comment:
Just an idea, not sure if worth it, a question like:
How urgent do you find your bug report to be?

MINOR
MODERATE
BREAKING
something along those lines? Could that immediately put some labels in the issues? That why we can quickly glance if some Triage issue is also reported as very urgent.

Reply:
We already have that kind of labels. In any case, we are the ones to assign them, not the users who report an issue.

Reply:
Right, it would still have the triage label and we would fix it, if we though the user incorrectly assigned the urgency, but it would give us an initial feeling for the priority of the issue with a quick glance.

Reply:
I've added more labels. We'll see how that works.

Reply:
I don't understand this. Doesn't this mean that the open issue will now have all these labels? The point is that the user would somehow select an option that would tie into a label? If that is not possible, then I'd just have the Bug & Triage labels set.
This way the issue created will just inherit all these labels that don't make sense, they should be mutually exclusive. Also Maybe "**Critical**" is a better naming then "**Breaking**", I just typed whatever I remembered at the time. And I thought you mentioned we already had those anyway, so just re-use?
labels: ["Type: Bug \U0001F41E", "Triage", "Minor", "Moderate", "Breaking"]